.asf.yaml

```diff
@@ -38,11 +38,15 @@ github:
   protected_branches:
     main:
       required_pull_request_reviews:
+        # if we want to use this, GitHub CODEOWNERS should be cleanly define, with different patterns/owners
         require_code_owner_reviews: false
         required_approving_review_count: 1
 
       required_linear_history: true
 
+      # requires all conversations to be resolved before merging
+      required_conversation_resolution: true
```

Comment:
TBH, I'd prefer a "soft rule".
Comments, on code line or not, can contain useful information for later - for example referenced in issues or elsewhere or just left on the PR for reference.
"Resolving" with this flag would mean that the commenter resolves this, which might not happen "just in time", although the issue is resolved.

Reply:
That's a good point. Happy to close this PR if majority is OK with "soft rule".

Reply:
+1 for a soft rule. Usually I keep important discussions in unresolved state, so that people can find it more easily.

Reply:
+1 for a soft rule as well.

Reply:
+1 to soft rule.

Reply:
I'm good with a soft rule